### DISCUSSION TITLE
Unknown User shown after publishing a package

### DISCUSSION THREAD

**Discussion by Nate-Wilkins** [verdaccio/verdaccio#2283] [Q&A] [answered]
**Describe the bug**

Unknown user for published package after logging in and publishing a scoped package.
Everything is default configuration on latest version (v5.1.0).

**To Reproduce**

__package.json__
```
  "publishConfig": {
    "registry": "http://localhost:4873"
  }
```

__Publish command:__
```
npm login
# username
# password
# email
npm publish
```

**Expected behavior**

Show listed package from the user who published the package.

**Screenshots**

![image](https://user-images.githubusercontent.com/2957868/120944170-e6fcbc00-c6e7-11eb-936f-afb7939543f8.png)

__Server log:__
```
 http --- 172.17.0.1 requested 'GET /-/verdaccio/packages'
 info --- auth/allow_action: access granted to: undefined
 info --- allowed access for @scope/some-package
 http --- 304, user: null(172.17.0.1), req: 'GET /-/verdaccio/packages', bytes: 0/0
```

**Environment information**

Running the docker container for verdaccio.

**Comment by juanpicado** [comment, marked as answer]
The `author` comes from the `package.json`, check if the content in that field has a correct format otherwise  fall back to what you see on the screen.
